bin/grommet.js

```diff
@@ -43,8 +43,8 @@ var tasks = {
         process.chdir('./' + app);
 
         var templateFolder = path.join(grommetPath, 'templates/init/**');
-        var mobileIcon = path.join(grommetPath, 'mobile-app-icon.png');
-        var shortcutIcon = path.join(grommetPath, 'shortcut-icon.png');
+        var mobileIcon = path.join(grommetPath, 'src/img/mobile-app-icon.png');
```

Comment:
<img width="1308" alt="screen shot 2016-06-27 at 5 47 08 pm" src="https://cloud.githubusercontent.com/assets/1207250/16400340/34ce9428-3c8f-11e6-8586-894d8368c330.png">

this is the content of the install grommet in your node_modules folder.